doc/developer-guide/api/functions/TSHttpTxnClientProtocolGet.en.rst

```diff
@@ -0,0 +1,34 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed
+   with this work for additional information regarding copyright
+   ownership.  The ASF licenses this file to you under the Apache
+   License, Version 2.0 (the "License"); you may not use this file
+   except in compliance with the License.  You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied.  See the License for the specific language governing
+   permissions and limitations under the License.
+
+.. include:: ../../../common.defs
+
+.. default-domain:: c
+
+TSHttpTxnClientProtocolGet
+*********************
+
+Gets the protocol string (http, http/2) of a specified transaction. 
+
+Synopsis
+========
+
+`#include <ts/ts.h>`
+
+.. function:: const char * TSHttpTxnClientProtocolGet(TSHttpTxn txnp)
+
+Description
+===========
```

Comment:
I wonder if we should bake this docs section into ./doc/developer-guide/api/functions/TSHttpTxnClientReqGet.en.rst ? We have too many man files already, and the intent was always to group API functions that are related into one file.

I'm not 100% TSHttpTxnClientProtocolGet belongs in that particular file above, but maybe you can take a look and see what, if any, APIs should be merged together? (Feel free to coalesce other files as well :).